test/presets/netlify.test.ts

```diff
@@ -137,3 +187,139 @@ describe("nitro:preset:netlify", async () => {
     }
   );
 });
+
+describe("nitro:preset:netlify-edge", async () => {
+  const publicDir = resolve(getPresetTmpDir("netlify-edge"), "dist");
+  const ctx = await setupTest("netlify-edge", {
+    config: {
+      framework: {
+        name: "mock-framework",
+        version: "1.2.3",
+      },
+      publicAssets: [
```

Comment:
This seems like duplicate logic we can at least share between two netlify/edge tests.

Ideally, I would love to extend the main fixture with more cases of public assets so that we increase overall coverage instead of per-preset tests.